org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/package.json

```diff
@@ -24,6 +25,8 @@
   "devDependencies": {
     "optimist": "*",
     "ava": "^0.15.2",
-    "es6-sleep": "*"
+    "es6-sleep": "*",
+    "opent2t-device-smartthingshub": "file:../../../org.opent2t.sample.hub.superpopular/com.smartthings.hub/test",
+    "opent2t-device-binaryswitch": "file:../../test"
```

Comment:
Is this getting the mock devices?

---
In reply to: [90383696](https://github.com/openT2T/translators/pull/135#discussion_r90383696) [](ancestors = 90383696)